### PR TITLE
Move logic to retrieve system uuid to detect_utils

### PR DESCRIPTION
**Pull request by elfosardo**
Being a generic helper function, get_uuid should be moved to
detect_utils.
This is also needed for the future refactor of detect_system.

### CODE REVIEW

File: hardware/detect_utils.py [hunk 1]
```diff
@@ -231,3 +232,42 @@ def clean_str(val):
 def clean_tuples(lst):
     """Clean a list of tuples from bad UTF-8 strings."""
     return [tuple([clean_str(val) for val in elt]) for elt in lst]
+
+
+def _get_uuid_x86_64():
+    """Get uuid from dmidecode"""
+
+    uuid_cmd = subprocess.Popen(
+        "dmidecode -t 1 | grep UUID | awk '{print $2}'", shell=True,
```
Comment (ErwanAliasr1) [inline hardware/detect_utils.py:241]: Can you take the opportunity to switch to "dmidecode -s system-uuid" ?

Comment (elfosardo) [reply hardware/detect_utils.py:241]: I'd rather do this in a different patch, I prefer keeping refactoring work and logic changes separated

Comment (ErwanAliasr1) [reply hardware/detect_utils.py:241]: ack

Comment (ErwanAliasr1) [reply hardware/detect_utils.py:241]: Can you rebase and I'll merge it.

Comment (elfosardo) [reply hardware/detect_utils.py:241]: will do, thanks!

**Pull request merged**